R/caretEnsemble.R

```diff
@@ -142,7 +141,15 @@ predict.caretEnsemble <- function(object, keepNA = TRUE, se = FALSE, return_weig
       message("Predictions being made only for cases with complete data")
     }
     est <- as.numeric(preds %*% object$weights)
-    se.tmp <- apply(preds, 1, FUN = wtd.sd, weights = object$weights, normwt = TRUE)
+    if(dim(preds)[1]<1000){ # use apply for small data
+      se.tmp <- apply(preds, 1, FUN = wtd.sd, weights = object$weights, normwt = TRUE)
+    } else { # switch to foreach for large data
+      require('foreach')
```

Comment:
Let's import foreach, rather than loading the entire package.  Remove this line and run `devtools::use_package('foreach')` instead.